dll/win32/msafd/msafd.h

```diff
@@ -559,7 +588,7 @@ MsafdReturnWithErrno(NTSTATUS Status,
     }
     else
     {
-        DbgPrint("%s: Received invalid lpErrno pointer!\n", __FUNCTION__);
+        //DbgPrint("%s: Received invalid lpErrno pointer!\n", __FUNCTION__);
```

Comment:
What's the reason for commenting that DbgPrint?

Reply:
Calling `MsafdReturnWithErrno` with `lpErrno == NULL` is valid in some cases.

Reply:
Then you can simply use `DPRINT()` macro instead, which can be muted when not needed, by `#define NDEBUG` at the top of the file (you need to add `#include <debug.h>` above that too):
```suggestion
        DPRINT("%s: Received invalid lpErrno pointer!\n", __FUNCTION__);
```

Reply:
Or no, better use `TRACE()` instead, since Wine debug channel is already included here.

Reply:
Removed as it's no longer considered an error.